### DISCUSSION TITLE
How to fetch additionnal posts in SSG?

### DISCUSSION THREAD

**Discussion by heyalexnet** [nuxt/content#1685] [Q&A]
Hello,
I have a page that list a few posts and a button to enable to see more posts.
Here is the link to a minimal case: https://stackblitz.com/edit/nuxt-starter-xbc4jj?file=app.vue

It works as intended in dev mode but I cannot make it work in SSG (`generate`, then `preview`): when I click on the button it doesn't show the additionnal posts.

What am I missing? Is there another way to do that that will work in SSG?

**Comment by stenet** [comment]
@ineation did you find a solution?

**Reply by heyalexnet** [reply]
I have used an horrible hack where I generate one page for each possibility (combination of category and pager). 🤫

**Comment by stenet** [comment]
@Atinux do you have an idea?

Currently I had to disable SSG to use `queryContent()` for runtime queries. This works but on Cloudflare Pages now every page view counts as a request.
